components/panel/style.scss

```diff
@@ -104,8 +104,15 @@
 	align-items: center;
 	margin-top: 20px;
 
+	select {
+		// Necessary for select to respond to flexbox
+		min-width: 0;
+	}
+
 	label {
 		margin-right: 10px;
+		flex-shrink: 0;
+		max-width: 75%;
```

Comment:
Was this necessary?

Reply:
It balances out the label width to keep it from wrapping when the select is long and label short - see this comparison: 

![label-fix](https://user-images.githubusercontent.com/445861/33571478-184ce0c2-d8fe-11e7-9c24-2328fe053385.png)

Note that the "After" screenshot above shows the label for "Stick to the front page" wrapping, but the max-width at 75% instead solves that.

Reply:
Would this have been possible to achieve with `white-space: nowrap`?

https://developer.mozilla.org/en-US/docs/Web/CSS/white-space

Reply:
For short label like this one, yes, but we wouldn't want to prevent wrapping for very long labels – with this solution, the label will wrap only when it starts to fill more than the max-width of 75%.